docs/docker.md

```diff
@@ -127,42 +68,36 @@ The postgres container will have created a database already, but we need
 to migrate the Django project to this database. The general way to run
 the manage.py command in docker is like this:
 
-    docker-compose -f dev.yml django python manage.py <manage command and options>
+    docker-compose exec -f dev.yml django python manage.py <manage command and options>
```

Comment:
@lauralorenz Can you please correct the order of the arguments in this command.  The `-f dev.yml` should come before the exec:
`docker-compose -f dev.yml exec django python manage.py <manage command and options>`